boudicca.events/eventcollectors/src/main/kotlin/events/boudicca/eventcollector/collectors/MetalCornerCollector.kt

```diff
@@ -0,0 +1,70 @@
+package events.boudicca.eventcollector.collectors
+
+import base.boudicca.SemanticKeys
+import base.boudicca.api.eventcollector.Fetcher
+import base.boudicca.api.eventcollector.TwoStepEventCollector
+import base.boudicca.model.Event
+import base.boudicca.model.EventCategory
+import org.jsoup.Jsoup
+import org.jsoup.nodes.Document
+import java.time.LocalDateTime
+import java.time.format.DateTimeFormatter
+import java.util.*
+
+class MetalCornerCollector : TwoStepEventCollector<Triple<String, String, Document>>("metalcorner") {
+
+  private val baseUrl = "https://www.escape-metalcorner.at/"
+
+  override fun getAllUnparsedEvents(): List<Triple<String, String, Document>> {
+    val fetcher = Fetcher()
+
+    val events = mutableListOf<Triple<String, String, Document>>()
+    val eventUrls = mutableListOf<Pair<String, String>>()
+
+    val document = Jsoup.parse(fetcher.fetchUrl(baseUrl + "de/events"))
+    document.select("div#content > div#events .event")
+      .forEach {
+        eventUrls.add(Pair(it.select(".head").text(), it.select("a.overlay").attr("href")))
+      }
+
+    eventUrls.forEach {
+      val url = baseUrl + it.second.substring(2)
+      val eventType = it.first
+      events.add(Triple(eventType, url, Jsoup.parse(fetcher.fetchUrl(url))))
+    }
+
+    return events
+  }
+
+  override fun parseEvent(event: Triple<String, String, Document>): Event {
+    val (eventType, url, doc) = event
+    val data = mutableMapOf<String, String>()
+
+    val name = "Escape - " + doc.select("div#content h1").text()
+    data[SemanticKeys.NAME] = name
```

Comment:
should we relly prefix the event name here?

Reply:
an event name from the event website is like "Violent Way, Die Infizierten, Bonfire Punks, Dajneseri" but the title of the event page is prefixed with "Escape - ", I thought if I follow the second one, it would make more sense.

Reply:
hmm... not sure about that
we do not have the location name in the event name anywhere else
i would remove it

Reply:
ok